src/draive/conversation/realtime/default.py

```
@@ -0,0 +1,188 @@
+) -> AsyncIterator[ConversationStreamElement]:
+    initial_context: Sequence[LMMContextElement] = tuple(
+        message.as_lmm_context_element()
+        for message in await memory.recall()
+        if isinstance(message, ConversationMessage)  # skip events in LMM context
+    )
+
+    input_queue = AsyncQueue[LMMStreamInput]()
+
+    async def process_input() -> None:
+        try:
+            # TODO: include guardrails check
```

Pylint (3.3.7)

[warning] TODO: include guardrails check

(W0511)